searchapp/views.py

```diff
@@ -55,9 +55,10 @@ def contact(request):
 
 
 def student_view(request):
+    paper_type = request.GET.get('type')
```

Comment:
Make `paper_type` as `worksheet_type`. The same to be changed anywhere where `paper_type` has been used.